modules/elt/pages/terraform-airbyte-provider.adoc

```diff
@@ -0,0 +1,222 @@
+
+== Configuring the variables.tf file
+
+[source, bash]
+----
+#log in to https://portal.airbyte.com generate, save and populate the variable with your API key
+variable "api_key" {
+    type = string
+    default = ""
+}
+#workspace_id is found in the url to your Airbyte Cloud account https://cloud.airbyte.com/workspaces/<workspace_id>/settings/dbt-cloud 
+variable "workspace_id" {
+    type = string
+    default = ""
+} 
+
+#Google spreadsheet id and Google private key
+variable "spreadsheet_id" {
+    type = string
+    default = ""
+}
+variable "google_private_key" {
+  type = string
+  default =  ""
+}
+# Teradata Vantage connection credentials
+variable "host" {
+  type = string
+  default = ""
+  }
+variable "username" {
+  type = string
+  default = "demo_user"
+  }
+  variable "password" {
+  type = string
+  default = ""
+  }
+----
+
+== Execution Commands
+
+Run `terraform init` pull down provider plugin from terraform provider page and initialize a working Terraform directory.
+
+This command should only be run after writing a new Terraform configuration or cloning an existing one from version control.
+
+image::{dir}/terraforminit.png[Initialize Terraform with Terraform init command, align="center" width=75%]
+
+Run `terraform plan` to display the execution plan Terraform will use to create resource and make modifications to infrastructure. 
+
+For this example you will a plan for 3 resources created:
+
+Connection: # airbyte_connection.googlesheets_teradata will be created
+
+Destination: # airbyte_connection.googlesheets_teradata will be created
+
+Source: # airbyte_source_google_sheets.my_source_gsheets will be created
+  
+image::{dir}/terraformplan.png[View Terraform execution plan with terraform plan command, align="center" width=75%]
+
+Run `terraform apply` and `yes` to generate a plan and carry out the plan.
+
+image::{dir}/terraformapply.png[Apply the Terraform plan with terraform apply command, align="center" width=75%]
+
+The `terraform.tfstate` file is created after you run `terraform apply` for the first time. This file tracks the status of all sources, destinations, and connections managed by Terraform. For subsequent executions of `Terraform apply`, Terraform compares the code in the `main.tf` file with the code stored in the `tfstate` file. If resources are added or removed in `main.tf`, Terraform automatically updates both your deployment and the `.tfstate` file accordingly upon deployment. Do not modify this file by hand.
+
+You now have a Source, Destination and Connection on Airbyte Cloud created and managed via Terraform. 
+
+image::{dir}/airbyteconnection.png[Airbyte Connection in Airbyte Cloud UI, align="center" width=75%]
```

Comment:
Are there other resources we can recommend, like the 2 other airbyte-related quickstarts?